data/hackathon/scenario7.mdx

```diff
@@ -40,9 +40,18 @@ Explain what you see when going through podman desktop.
 Demonstrate the ACME team how you have out of the box recipes you can leverage.
 - Use the Summarizer recipe, change the code such that it shows a welcome message with your team name in it.
 - Take a screenshot and post into the slack channel.
+
+7.4 - Check your work
+
 ![Alt text](/hackathon/static/images/hackathon/summarizerHello.png "You want to show something like that plus a summarized document")
-                  
 
+If you got your text summarization working on a document of your choice then pls
+post a screenshot of the app showing your team name in the welcome message and the summarizer working to #event-anz-ocp-ai-hackathon with the message:
+
+Please review [team name] solution for exercise 7.
+
+This exercise is worth 750k. If you engage the services of the contractor (by obtaining a hint from the hackathon moderators), it will cost you 25k, which will be deducted from the respective challenge's deal value to your team. The event team will reply in slack to confirm your updated team total deal size.
```

Comment:
```suggestion
This exercise is worth `750k`. If you engage the services of the contractor (by obtaining a hint from the hackathon moderators), it will cost you `25k`, which will be deducted from the respective challenge's deal value to your team. The event team will reply in slack to confirm your updated team total deal size.
```